src/gebt_poc/solver.cpp

```diff
@@ -468,28 +468,28 @@ void ConstraintsResidualVector(
 ) {
     auto translation_0 = Kokkos::subview(gen_coords, Kokkos::make_pair(0, 3));
     auto rotation_0 = Kokkos::subview(gen_coords, Kokkos::make_pair(3, 7));
```

Comment:
These variables appear unused.  Either use them instead of gen_coords in the lambda or delete them here.

Reply:
Removed